ydr/operators.py

```diff
@@ -99,7 +99,7 @@ def convert_separate_drawables(
                 drawable_obj = convert_obj_to_drawable(obj)
 
                 if auto_embed_col:
-                    composite_obj = convert_obj_to_composite(duplicate_object(obj), SollumType.BOUND_GEOMETRYBVH, True)
+                    composite_obj = convert_obj_to_composite(duplicate_object(obj), SollumType.BOUND_GEOMETRYBVH, True, do_center=bool)
```

Comment:
Should this be `do_center=True`? As it is, it doesn't make sense (you are passing a `class` not a `True`/`False` value) but it coerces to `True`.

Reply:
Yes, will fix this